src/ApiFunction.ts

```diff
@@ -47,21 +47,20 @@ export class ApiFunction extends Construct {
    * Monitor the logs generated by this function for a filter pattern, generate metric
    * and alarm on increased error rate.
    *
-   * @param monitoredBy Lambda function responsible for monitoring this function
    * @param filterPattern Pattern to filter by (default: containing ERROR)
    */
   private monitor(filterPattern?: IFilterPattern) {
-    const filter = new MetricFilter(this, 'MetricFilter', {
+    const errorMetricFilter = new MetricFilter(this, 'MetricFilter', {
```

Comment:
Hier ook.

Reply:
Dit is een wijziging van september die om een of andere reden weer terugkomt. De naam is hier errorMetricFilter (hernoemd van 'filter'), om dezelfde reden. Wordt alleen in deze functie gebruikt, functie is 15 regels of zo. Specifieker is lastig, want wordt door verschillende lambdas gebruikt, en het is een filter op errors, dus op zich wel beschrijvend?